client-src/elements/chromedash-admin-blink-page.ts

```diff
@@ -62,29 +69,21 @@ export class ChromedashAdminBlinkPage extends LitElement {
       `,
     ];
   }
-  static get properties() {
-    return {
-      loading: {type: Boolean},
-      user: {type: Object},
-      _editMode: {type: Boolean},
-    };
-  }
-
-  /** @type {import('chromestatus-openapi').DefaultApiInterface} */
-  _client;
 
-  /** @type {Array<import('chromestatus-openapi').OwnersAndSubscribersOfComponent>} */
+  @property({type: Boolean})
+  loading = true;
+  @property({type: Object})
+  user = null;
+  @property({type: Boolean})
+  _editMode = false;
+  @property({
+    type: Array<OwnersAndSubscribersOfComponent>,
+  })
   components;
```

Comment:
This syntax doesn't work to tell Typescript what type a field has: https://lit.dev/docs/components/properties/#property-options:~:text=reactive%20state.-,type,-When%20converting%20a

The VSCode `runem.lit-plugin` extension notices this and underlines the incorrect type in yellow. Do you have that installed?

You'll want
```suggestion
  @state()
  components: Array<OwnersAndSubscribersOfComponent> | undefined;
```

(I changed `@property({attribute:false})` to `@state()` because these fields are assigned in `fetchData()` rather than in a surrounding template.)

Reply:
this results in a lot of `object might be undefined` errors. I manually added ! after each but not sure if there's a better way to do so

Reply:
I think the `!`s are correct for this, at least for this change. We might want to add some more explicit conditions to handle when `listComponentUsers` returns an undefined `components` field, or it might be that https://github.com/GoogleChrome/chromium-dashboard/blob/1b192c0ecfbb4f0e1eda57c4387bd9b7df5c8bbb/openapi/api.yaml#L192-L201 should change to list its properties as required (@jcscottiii?), but we can do either of those in a future PR.

Reply:
We can definitely do that given that those two keys are always returned

https://github.com/GoogleChrome/chromium-dashboard/blob/1b192c0ecfbb4f0e1eda57c4387bd9b7df5c8bbb/api/components_users.py#L63-L64

Reply:
And the syntax for that is in https://swagger.io/docs/specification/data-models/dictionaries/#fixed-keys / https://json-schema.org/learn/getting-started-step-by-step#define-required-properties.